doc/user/content/quickstarts/apps.md

```diff
@@ -0,0 +1,339 @@
+---
+title: "Data Applications"
+description: "Data Applications"
+menu:
+  main:
+    parent: 'quickstarts'
+    weight: 30
+    name: 'Data Applications'
+aliases:
+  - /demos/data-applications
+---
+
+In this quickstart, you'll build the analytics for an eCommerce sales dashboard that queries Materialize.
+At the end of this guide, you'll have the chance to explore an eCommerce data application querying against your code.
+
+The key concepts present in this quickstart will also apply to many other projects:
+
+* [Sources](https://materialize.com/docs/sql/create-source/load-generator/)
+* [Materialized views](https://materialize.com/docs/sql/create-materialized-view/)
+* [Subscriptions](https://materialize.com/docs/sql/subscribe/)
+
+### Prepare the environment
+
+1. Set up a [Materialize account.](/register)
+
+1. [Install psql and connect](https://materialize.com/docs/get-started/#connect) to Materialize.
+
+### Create the sources
+
+Materialize provides public Kafka topics for its users to use as a source. The Kafka topics contain data from a fictional eCommerce and receive updates every second. You will use the data in them to build the analytics.
+
+1. In your `psql` terminal, create a new schema:
+
+    ```sql
+    CREATE SCHEMA shop;
+    ```
+1. Create the connection:
+
+    ```sql
+    CREATE CONNECTION shop.kafka_conn TO KAFKA (BROKER 'localhost:9092');
```

Comment:
I think that we could also mention the confluent schema registry connection here as well.

Reply:
Correct too!